python/cli/prelude_cli/views/build.py

```diff
@@ -97,8 +97,10 @@ def test_id():
 
     def upload(p: Path):
         with open(p, 'r') as source_code:
-            controller.upload(test_id=identifier, filename=p.name, code=source_code.read())
+            out = controller.upload(test_id=identifier, filename=p.name, code=source_code.read())
             click.secho(f'Uploaded {path}', fg='green')
+            click.secho(f'Compiled {path}', fg='green')
```

Comment:
let's kill the secho here -- I'm aiming to make the CLI only JSON, no other print outs

Reply:
done